realm/functions/get_token/source.js

```diff
@@ -2,46 +2,63 @@ exports = async function () {
   const GOOGLE_TOKEN_ENDPOINT = "https://oauth2.googleapis.com/token";
```

Comment:
Since this is a YouTube specific function, may want to indicate that in the name or in a comment

Reply:
It will support multiple auth providers eventually, and these constants will probably be stored either in some sort of dictionary in the code, or in the database.